### DISCUSSION TITLE
Demo Gif

### DISCUSSION THREAD

**Discussion by mrjones2014** [mrjones2014/legendary.nvim#88] [Show and tell]
![CleanShot-2022-03-25-at-07 27 26](https://user-images.githubusercontent.com/8648891/160112850-5fbbf327-309c-4bac-ad6b-df217127d886.gif)

**Comment by mrjones2014** [comment]
updated version



![demo](https://user-images.githubusercontent.com/8648891/200827633-7009f5f3-e126-491c-88bd-73a0287978c4.gif)

**Reply by technicalpickles** [reply]
Would you mind sharing the configuration you've used? I'm using dressing with telescope, and it definitely looks different.

**Reply by mrjones2014** [reply]
Sure. 

Dressing.nvim config: https://github.com/mrjones2014/dotfiles/blob/master/.config/nvim/lua/my/configure/dressing.lua

Telescope.nvim config: https://github.com/mrjones2014/dotfiles/blob/master/.config/nvim/lua/my/configure/telescope.lua

Legendary.nvim config: https://github.com/mrjones2014/dotfiles/blob/master/.config/nvim/lua/my/configure/mine/legendary.lua
